intermediate_source/parametrizations.py

```diff
@@ -255,7 +255,7 @@ def forward(self, X):
 parametrize.register_parametrization(layer_spd, "weight", MatrixExponential())
 X = layer_spd.weight
 print(torch.dist(X, X.T))                        # X is symmetric
-print((torch.symeig(X).eigenvalues > 0.).all())  # X is positive definite
+print((torch.linalg.eigvalsh(X, UPLO="U") > 0.).all())  # X is positive definite
```

Comment:
Write simply
```suggestion
print((torch.linalg.eigvalsh(X) > 0.).all())  # X is positive definite
```
as `X` is symmetric.